### ISSUE TITLE
Mark pre release maps with disclaimer

### ISSUE THREAD

**Issue by marcoabreu** [Team-OZE/LegionTD-Map#13]
I think we should add a disclaimer inside the game as well as in the title and description of the map that an auto generated pre-release is an unofficial pre-release which is not intended for public play. 

The pipeline should detect the release mode and add the info dynamically.

**Comment by r-mach** [comment]
Indeed if we want to avoid weird artifacts being spread on bnet.

I was thinking in making a specific "release" pipeline and parsing/replacing in `MapData` version accordingly.
Something like `$$OZ_VERSION$$` could be default to `TEST MAP` and when tagged, put the correct version number like `v5.0`.

**Comment by marcoabreu** [comment]
Yeah, plus we could add the commit hash to the map somewhere so we know which version we got
